### PR TITLE
Add a greeting

### PR DESCRIPTION
**Pull request by amexabee**
A greeting file

Add a home page
Link a CSS file to the HTML file
Add linters

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+header {
```
Comment (Nyame-Wolf) [inline styles.css]: Please add a class to the `<h1>` element. This is part of the requirements
Screenshot 2022-09-21 at 14 49 01

File: README.md [hunk 2]
```diff
@@ -1 +1 @@
-# hello
```
Comment (Nyame-Wolf) [inline README.md:1]: Kindly note that you are required to use the [Microverse Readme template] as per the project requirements. (https://github.com/microverseinc/readme-template/blob/master/README.md) as per the requirements
Screenshot 2022-09-22 at 13 45 59

- Note that the read me explains to your users how the app works works.
- Also note that for subsequent projects, this is a cause for invalidation of your submission

**Review by Nyame-Wolf — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle: 

Hi @amexabee  :wave: 

Your project page is looking fantastic so far :star_struck: ! 

However, there is still some room for improvement, that can help you have better code and make your project professional :sunglasses: 



## Highlights :sparkles:

- Great work using the right GitHub Flow :100: 
- Commits are descriptive and meaningful :+1: 
- tasks implementation correct.


Congrats on your work so far! :clap:  

## Required Changes ♻️

- [ ] Kindly note that as per the requirements you were required to set up the [html/css linters](https://github.com/microverseinc/linters-config/tree/master/html-css). This assist to get rid of simple syntax errors. **NOTE** that for subsequent code reviews, you submission will be invalidated

- [ ] Kindly add a descriptive pull request message. This may lead to pull request invalidation next time. I would like to however commend your descriptive title.

There are some issues that you still need to work on to go to the next project, but you are almost there! 
_For more information, check the comments under the review :arrow_down:_

## Optional suggestions :ballot_box_with_check: 

_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code :100:_

Happy coding and keep it up! :raised_hands: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
  Please, remember to tag me **@Nyame-Wolf**  in your question so I can receive the notification.

  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

  _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by amexabee** [comment]
Thank you for your reply. I got everything that must be fixed except the last one. I am not sure what to change because I tried to follow the Microverse Readme template.
    On Thursday, September 22, 2022 at 04:35:16 PM GMT+3, Mumenya Nyamu ***@***.***> wrote:  
 
 


@Nyame-Wolf requested changes on this pull request.

STATUS: REQUIRED CHANGES ♻️
 
Hi @amexabee 👋
 
Your project page is looking fantastic so far 🤩 !

However, there is still some room for improvement, that can help you have better code and make your project professional 😎
 


Highlights ✨
    
   - Great work using the right GitHub Flow 💯
   - Commits are descriptive and meaningful 👍
   - tasks implementation correct.

Congrats on your work so far! 👏
 
Required Changes ♻️
    
   -    
 Kindly note that as per the requirements you were required to set up the html/css linters. This assist to get rid of simple syntax errors. NOTE that for subsequent code reviews, you submission will be invalidated

   -    
 Kindly add a descriptive pull request message. This may lead to pull request invalidation next time. I would like to however commend your descriptive title.


There are some issues that you still need to work on to go to the next project, but you are almost there!
For more information, check the comments under the review ⬇️
 
Optional suggestions ☑️
 
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code 💯
 
Happy coding and keep it up! 🙌
 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Nyame-Wolf in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

_As described in the Code reviews limits policyyou have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form

In styles.css:
> @@ -0,0 +1,3 @@
+header {

Please add a class to the <h1> element. This is part of the requirements


In README.md:
> @@ -1 +1 @@
-# hello

Kindly note that you are required to use the [Microverse Readme template] as per the project requirements. (https://github.com/microverseinc/readme-template/blob/master/README.md) as per the requirements

   
   - Note that the read me explains to your users how the app works works.
   - Also note that for subsequent projects, this is a cause for invalidation of your submission

File: README.md (hunk 2 above)
Comment (Nyame-Wolf) [reply README.md:1]: Hi @amexabee 
If you are referring to  the descriptive pull request, your pull request needs to be descriptive
Screenshot 2022-09-22 at 18 16 15
 
click on the tree dots highlighted above to include a more descriptive pull request description

**Comment by amexabee** [comment]
"Kindly note that you are required to use the [Microverse Readme template] as per the project requirements. (https://github.com/microverseinc/readme-template/blob/master/README.md) as per the requirements"
    On Thursday, September 22, 2022 at 06:13:01 PM GMT+3, Mumenya Nyamu ***@***.***> wrote:  
 
 


@Nyame-Wolf commented on this pull request.

In README.md:
> @@ -1 +1 @@
-# hello

Hi @amexabee
which is the last one you refer to?

**Review by mckent05 — APPROVED** [review]
Hi @amexabee, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mgmediaweb — COMMENTED** [review]
Hi @amexabee , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

work4

## Highlights 💯 
- Professional-looking readme file. ✔️ 
- Clear and descriptive Pull Request. ✔️ 

## Required Changes ♻️

Kindly, you presented the first work "hello microverse", you in this exercise must start with your portfolio, choose a design from those suggested in Figma, and make the first cover section, review the project requirements and resubmit this same PR with the required changes, good luck

image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**